### PR TITLE
Add Tests for AddProgramCommandTest

### PR DESCRIPTION
**Pull request by branzuelajohn**

### CODE REVIEW

File: src/test/java/dog/pawbook/logic/commands/AddCommandIntegrationTest.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public void execute_newOwner_success() {
     }
 
     @Test
-    public void execute_duplicateOwner_throwsCommandException() {
+    public void execute_Owner_throwsCommandException() {
```
Comment (kouyk) [inline src/test/java/dog/pawbook/logic/commands/AddCommandIntegrationTest.java]: why is this test method renamed? isn't it correct?

File: src/test/java/dog/pawbook/logic/commands/DeleteProgramCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package dog.pawbook.logic.commands;
+
+public class DeleteProgramCommandTest {
+}
```
Comment (kouyk) [inline src/test/java/dog/pawbook/logic/commands/DeleteProgramCommandTest.java:4]: leave out the file if not written so others can easily tell if the test has been written

File: src/test/java/dog/pawbook/testutil/TypicalEntities.java [hunk 3]
```diff
@@ -73,6 +74,10 @@
     public static final Dog GENIE = new DogBuilder().withName("Genie").withSex("male")
             .withDateOfBirth("29-05-2020").withBreed("Husky").withOwnerID(13).build();
 
+    // Programs
+    public static final Program PROGRAM1 = new ProgramBuilder().withName("Basic Obedience Training")
+            .withDogs(1).withSessions("02-02-2020 18:00").withTags("puppies").build();
+
```
Comment (kouyk) [inline src/test/java/dog/pawbook/testutil/TypicalEntities.java:80]: this isn't used anywhere, I think it is better to leave it out and let the person who is writing program tests to everything

**Comment by branzuelajohn** [comment]
@kouyk done with the  changes

**Review by wei-yutong — APPROVED** [review]
lgtm :)

**Pull request merged**